### PR TITLE
Update Javadoc of premium SDK for azure-resourcemanager-compute

### PR DESCRIPTION
**Pull request by v-hongli1**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/ComputeManager.java [hunk 1]
```diff
@@ -151,23 +163,35 @@ private ComputeManager(HttpPipeline httpPipeline, AzureProfile profile) {
...
     public VirtualMachines virtualMachines() {
         if (virtualMachines == null) {
             virtualMachines = new VirtualMachinesImpl(this, storageManager, networkManager, authorizationManager);
         }
         return virtualMachines;
     }
 
-    /** @return the virtual machine image resource management API entry point */
+    /**
+     * Gest the virtual machine image resource management API entry point.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/ComputeManager.java:191]: Gets

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/ComputeManager.java]: Fixed by the [commit#68694ef](https://github.com/Azure/azure-sdk-for-java/pull/43499/commits/68694ef61ed373cc60dd24b9ea46f77420201faf)

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/CreationSource.java [hunk 2]
```diff
@@ -52,7 +56,11 @@ public CreationSourceType type() {
         return CreationSourceType.UNKNOWN;
     }
 
-    /** @return ID of the source */
+    /**
+     * Gets ID of the resource.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/CreationSource.java:60]:
```suggestion
     * Gets ID of the source.
```

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/Disk.java [hunk 3]
```diff
@@ -21,40 +21,82 @@
...
+    /**
+     * Gets the disk creation method.
+     *
+     * @return the disk creation method
+     */
     DiskCreateOption creationMethod();
 
-    /** @return true if the disk is attached to a virtual machine, otherwise false */
+    /**
+     * Whether the disk is attached to a virtual machine.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/Disk.java:39]: Checks ...

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/GalleryImage.java [hunk 4]
```diff
@@ -28,64 +28,144 @@
...
+     *
+     * @return the date indicating image's end of life.
+     */
     OffsetDateTime endOfLifeDate();
 
-    /** @return the image eula. */
+    /**
+     * Gets the image eula.
+     *
+     * @return the image eula.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/GalleryImage.java:62]: nit, image EULA

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/GalleryImage.java [hunk 5]
```diff
@@ -28,64 +28,144 @@
...
+     *
+     * @return the provisioningState of image resource.
+     */
     String provisioningState();
 
-    /** @return the purchasePlan of the image. */
+    /**
+     * Gets the purchasePlan of the image.
+     *
+     * @return the purchasePlan of the image.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/GalleryImage.java:125]: nit, purchase plan

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/GalleryImageVersion.java [hunk 6]
```diff
@@ -22,43 +22,90 @@
...
+    /**
+     * Gets the image version name.
+     *
+     * @return the image version name.
+     */
     String name();
 
-    /** @return the provisioningState of image version resource. */
+    /**
+     * Gets the provisioningState if image version resource.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/GalleryImageVersion.java:47]:
```suggestion
     * Gets the provisioningState of image version resource.
```

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/Disk.java (hunk 3 above)
Comment (weidongxu-microsoft) [reply sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/Disk.java:39]: Suggest do a search on `* Whether ` in this PR, and fix all you find.

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java [hunk 7]
```diff
@@ -251,37 +254,81 @@ Mono<RunCommandResult> runShellScriptAsync(List<String> scriptLines,
...
+    /**
+     * Gets the storage account type of the managed disk backing OS disk.
+     *
+     * @return the storage account type of the managed disk backing OS disk
+     */
     StorageAccountTypes osDiskStorageAccountType();
 
-    /** @return resource ID of the managed disk backing the OS disk */
+    /**
+     * GEts resource ID of the managed disk backing the OS disk.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java:314]: Gets

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineCustomImage.java [hunk 8]
```diff
@@ -19,19 +19,39 @@
...
+     *
+     * @return the hyper v Generation
+     */
     HyperVGenerationTypes hyperVGeneration();
 
-    /** @return ID of the virtual machine if this image was created by capturing that virtual machine */
+    /**
+     * Gets ID of the virtual machine.
+     *
+     * @return ID of the virtual machine if this image was created by capturing that virtual machine
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineCustomImage.java:39]: Use the full description in the `@return` in description. "Gets ID of the virtual machine." is not complete.

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineDiskOptions.java [hunk 9]
```diff
@@ -15,41 +15,67 @@ public final class VirtualMachineDiskOptions {
...
+     *
+     * @return the disk delete options
+     */
     public DeleteOptions deleteOptions() {
         return deleteOptions;
     }
 
     /**
-     * Gets whether the write accelerator is enabled. {@literal null} if no change.
+     * Whether the write accelerator is enabled. {@literal null} if no change.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineDiskOptions.java:56]: revert this

File: sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineDiskOptions.java [hunk 10]
```diff
@@ -15,41 +15,67 @@ public final class VirtualMachineDiskOptions {
...
      * @return whether the write accelerator is enabled
      */
     public Boolean writeAcceleratorEnabled() {
         return writeAcceleratorEnabled;
     }
 
-    /** @return whether disk encryption set is configured,
-     * either as the ID of disk encryption set, or as {@code null} to override default configuration. */
+    /**
+     * Whether disk encryption set is configured.
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineDiskOptions.java:65]: Checks

Comment (v-hongli1) [reply sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineDiskOptions.java]: Fixed by the [commit#6d08161](https://github.com/Azure/azure-sdk-for-java/pull/43499/commits/6d0816182ab673e2c8f296b4c2f6c0ba843f2867) and [commit#2570846](https://github.com/Azure/azure-sdk-for-java/pull/43499/commits/25708467caa912ea75c4cdbfe2e9e1f06b7bee25)

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**